.husky/pre-commit

```diff
@@ -3,3 +3,14 @@
 
 yarn lint-staged
 yarn type:check
+
+# this automatically builds plugin js files if you forgor
+! git diff HEAD --name-only | grep plugin
+if [ $? -eq 1 ]; then
```

Comment:
Is it recommended to use return code instead of embedding the command here directly?

Reply:
People have different [opinions](https://unix.stackexchange.com/questions/152966/should-i-embed-a-command-in-a-conditional-or-test-its-return-code) but I think it's more transparent in this form.